packages/@okta/vuepress-site/docs/guides/refresh-tokens/refresh-token-rotation/index.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Refresh token rotation
+---
+Refresh token rotation is an <ApiLifecycle access="ea" /> feature.
+
+Refresh token rotation helps a public client to securely rotate refresh tokens after each use. When refresh token rotation behavior is enabled in Okta, a new refresh token is returned each time the client makes a request to exchange a refresh token for a new access token.
+
+## Refresh token reuse detection
+
+When a client wants to renew an access token, it sends the refresh token with the access token request to the `/token` endpoint. Okta validates the incoming refresh token and issues a new set of tokens. As soon as the new tokens are issued, Okta invalidates the refresh token that was passed with the initial request to the `/token` endpoint.
+
+If a previously used refresh token is used again with the token request, the Authorization Server automatically detects the attempted reuse of the refresh token. As a result, Okta immediately invalidates the most recently issued access and refresh token. This protects your application from token compromise and replay attacks.
```

Comment:
`Okta immediately invalidates the most recently issued access and refresh token.` -> `Okta immediately invalidates the most recently issued refresh token and all access tokens from the time the user authenticated.`